php/php.dbgp/src/org/netbeans/modules/php/dbgp/packets/StackGetResponse.java

```diff
@@ -96,7 +96,7 @@ private void updateBreakpointsView(DebugSession session, List<Stack> stacks) {
         IDESessionBridge bridge = session.getBridge();
         if (bridge != null) {
             BreakpointModel breakpointModel = bridge.getBreakpointModel();
-            if (breakpointModel != null) {
+            if (breakpointModel != null && !breakpointModel.isSearchCurrentBreakpointById()) {
```

Comment:
If I am not wrong, `breakpointModel.isSearchCurrentBreakpointById()` can produce a NPE, right?